sdk/src/utils/resolvers.ts

```diff
@@ -0,0 +1,52 @@
+import { Resolvers, MeshContext } from "../../.graphclient";
+
+export const resolvers: Resolvers = {
+  Claim: {
+    chainName: (root, args, context, info) => context.chainName || "hypercerts-testnet", // The value we provide in the config
+  },
+  ClaimToken: {
+    chainName: (root, args, context, info) => context.chainName || "hypercerts-testnet", // The value we provide in the config
+  },
+  // Query: {
```

Comment:
Is this commented section still useful?

Reply:
Depend whether the solution works but I'm not sure what to expect from the app. Should a minted claim show up somewhere?